test/src/test/java/hudson/model/PasswordParameterDefinitionTest.java

```diff
@@ -82,6 +82,7 @@
 
         // Another control case: anyone can enter a different value.
         HtmlForm form = wc.withBasicApiToken(dev).getPage(p, "build?delay=0sec").getFormByName("parameters");
+        form.getElementsByAttribute("input", "class", "hidden-password-update-btn").get(0).click();
```

Comment:
This should probably use `HtmlElementUtil.click()` instead. Same for the other one below.